qemu/tests/cfg/balloon_service.cfg

```diff
@@ -1,32 +1,39 @@
 - balloon_service: install setup image_copy unattended_install.cdrom
     virt_test_type = qemu
     type = balloon_service
-    only Windows
+    no Win2000, Fedora.8, Fedora.9, Fedora.10, RHEL.3, RHEL.4, Unix, livecd
     no Host_RHEL.m5, Host_RHEL.m6
     monitor_type = "qmp"
     monitors = qmp1
     balloon = balloon0
     balloon_dev_devid = balloon0
     balloon_dev_add_bus = yes
-    driver_name = "balloon"
-    cdroms = "cd1 winutils virtio"
-    drive_index_virtio = 3
-    cdrom_virtio = isos/windows/virtio-win-1.8.0-4.iso
-    # balloon service related cmd, which will be covered by different
-    # guest's paths in guest-os cfg.
-    install_balloon_service = "%s:\Balloon\GUEST_OS\amd64\blnsvr.exe -i"
-    uninstall_balloon_service = "%s:\Balloon\GUEST_OS\amd64\blnsvr.exe -u"
-    status_balloon_service = "%s:\Balloon\GUEST_OS\amd64\blnsvr.exe status"
-    run_balloon_service = "%s:\Balloon\GUEST_OS\amd64\blnsvr.exe -r"
+    Windows:
+        driver_name = "balloon"
+        cdroms = "cd1 winutils virtio"
+        cdrom_virtio = isos/windows/virtio-win-1.8.0-4.iso
+        # balloon service related cmd, which will be covered by different
+        # guest's paths in guest-os cfg.
+        install_balloon_service = "%s:\Balloon\GUEST_OS\amd64\blnsvr.exe -i"
+        uninstall_balloon_service = "%s:\Balloon\GUEST_OS\amd64\blnsvr.exe -u"
+        status_balloon_service = "%s:\Balloon\GUEST_OS\amd64\blnsvr.exe status"
+        run_balloon_service = "%s:\Balloon\GUEST_OS\amd64\blnsvr.exe -r"
     repeat_times = 5
     base_path = "/machine/peripheral/"
     set_balloon_property = "guest-stats-polling-interval"
     get_balloon_property = "guest-stats"
-    polling_interval = 2
-    polling_sleep_time = 10
-    free_mem_cmd = wmic os get FreePhysicalMemory
+    polling_sleep_time = 20
     ratio = 0.1
     run_sub_test_after_balloon = no
     test_tags = "evict enlarge"
     balloon_type_evict = evict
     balloon_type_enlarge = enlarge
+    variants:
+        - small_polling_interval:
+            polling_interval = 2
+        - large_polling_interval:
+            polling_interval = 21474836
+            # No need to check the memory if set a large polling interval
+            # since qmp cmd 'qom-get' output for the balloon device won't
+            # change for a long time
+            mem_check = False
```

Comment:
@CongLi Yes, we could not wait for 21474836 seconds to check the status in our test. so just check if it will effect the following balloon evict/enlarge function.